modules/web/web-app/hooks/useZodForm.tsx

```diff
@@ -18,7 +18,7 @@ export function useZodForm<TSchema extends z.ZodTypeAny>(
     name,
   })
   const withError = (name: FieldPath<z.TypeOf<TSchema>>) => ({
-    error: form.getFieldState(name)?.error?.message,
+    error: form.formState.errors[name]?.message,
```

Comment:
an error in the form was not always triggering a rerender of the form components using `getFieldState`